.github/workflows/python.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8]
```

Comment:
@gzpcho do we still have 3.7 use case?

Reply:
Updated to `3.6` for consistency. Also specified the Ubuntu version since latest didn't support 3.6